src/Notes/WC_Admin_Note.php

```diff
@@ -504,6 +537,33 @@ public function clear_actions() {
 		$this->set_prop( 'actions', array() );
 	}
 
+	/**
+	 * Set note layout.
+	 *
+	 * @param string $layout Note layout.
+	 */
+	public function set_layout( $layout ) {
+		$this->set_prop( 'layout', $layout );
```

Comment:
Wondering if we should check to make sure the layout being set here is a valid one?

Reply:
Good idea! I added a check here in the commit [88f37ed](https://github.com/woocommerce/woocommerce-admin/pull/4262/commits/88f37ed0e31221c64e900116a8acb22d4462cfde)